Makefile

```diff
@@ -2032,8 +2032,8 @@ openstack_kuttl_run: ## runs kuttl tests for the openstack operator, assumes tha
 openstack_kuttl: export NAMESPACE = ${OPENSTACK_KUTTL_NAMESPACE}
 openstack_kuttl: input deploy_cleanup openstack openstack_deploy_prep ## runs kuttl tests for the openstack operator. Installs openstack operator and cleans up previous deployments before running the tests, cleans up after running the tests.
 	# Wait until OLM installs openstack CRDs
-	timeout $(TIMEOUT) bash -c "while ! (oc get crd | grep openstack.org); do sleep 1; done"
-	bash -c '(oc get crd openstacks.operator.openstack.org && make openstack_init) || true'
```

Comment:
I agree with removing this `|| true`.  We have seen it mask errors in CI.

Reply:
I think I remember now why the `|| true` was added in the first place.  The idea was to make this `openstack_kuttl` target backwards-compatible with older versions of OpenStack operator that did not have the new `OpenStack` initialization CRD.  i.e. the `oc get crd openstacks.operator.openstack.org` would return exit code 1 and thus be considered an error, so without the `|| true` the command would fail with the older OpenStack operator version and prevent the target from running.

The question becomes: do we need this target to work with older version of the OpenStack operator anymore?  The `main` branch of the OpenStack operator repo now has the `OpenStack` CRD.  Should the `main` branch of `install_yamls` be forced to assume it will always be working with this new install paradigm?

Reply:
I think the main branch can be considered for FR2 right. We already have 18.0-fr1

Reply:
thanks, @abays  that is simpler, done